assets/models/foundrylocal/qwen2.5-1.5b-instruct-test-openvino-npu/spec.yaml

```diff
@@ -0,0 +1,32 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: qwen2.5-1.5b-instruct-test-openvino-npu
+version: 1
+path: ./
+tags:
+  foundryLocal: "test"
+  license: "apache-2.0"
+  licenseDescription: "This model is provided under the License Terms available at <https://huggingface.co/Qwen/Qwen2.5-1.5B-Instruct/blob/main/LICENSE>."
+  author: Microsoft
+  inputModalities: "text"
+  outputModalities: "text"
+  task: chat-completion
+  maxOutputTokens: 2048
+  alias:
```

Comment:
```suggestion
  alias: qwen2.5-1.5b-instruct-test-openvino-npu
```

Reply:
Since we're unclear on whether `alias` is required, let's just make alias the same as model ID - should get ahead of any issues (hopefully).